EasyPost.Tests/ServicesTests/Beta/RateServiceTest.cs

```diff
@@ -0,0 +1,85 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using EasyPost.Models.API.Beta;
+using EasyPost.Tests._Utilities;
+using EasyPost.Tests._Utilities.Annotations;
+using EasyPost.Utilities;
+using EasyPost.Utilities.Annotations;
+using Xunit;
+
+namespace EasyPost.Tests.ServicesTests.Beta
+{
+    public class RateServiceTests : UnitTest
+    {
+        public RateServiceTests() : base("beta_rate_service")
+        {
+        }
+
+        #region Tests
+
+        #region Test CRUD Operations
+
+        [Fact]
+        [CrudOperations.Read]
+        [Testing.Function]
+        public async Task TestRetrieve()
+        {
+            UseVCR("retrieve");
+
+            Dictionary<string, object> data = Fixtures.BasicShipment;
```

Comment:
`data` is too ambiguous, please describe what the data is in the var. `params` or `shipmentData` are good options. This is true for all the `data` vars in this file.

```suggestion
            Dictionary<string, object> shipmentData = Fixtures.BasicShipment;
```